### PR TITLE
feat(client,server): basic database management views

### PR DESCRIPTION
**Pull request by Akirtovskis**
To test - create db and navigate to different views in db management section.

### CODE REVIEW

File: server/src/graphql/mutations/createDatabase.ts [hunk 1]
```diff
@@ -24,29 +25,25 @@ export const createDatabase: MutationResolvers['createDatabase'] = async (
   }
 
   // TODO implement other db support
-  if (input.type !== 'POSTGRESQL') {
```
Comment (pradel) [inline server/src/graphql/mutations/createDatabase.ts:27]: we still need this check until we support more db right?

File: client/src/ui/components/Spinner.tsx [hunk 2]
```diff
@@ -30,6 +30,7 @@ export const Spinner = ({ size, className, ...props }: SpinnerProps) => (
       {
         'border-8 border-t-8 h-24 w-24': size === 'large',
         'border-4 border-t-4 h-8 w-8': size === 'small',
+        'mt-1 border-2 border-t-2 h-4 w-4': size === 'extraSmall',
```
Comment (pradel) [inline client/src/ui/components/Spinner.tsx:33]: mt-1 should probably be removed from here, the other sizes don't have any margin

File: client/src/pages/database/env.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  }
+
+  return (
+    <div>
+      <Header />
+      <div className="max-w-5xl mx-auto px-4 sm:px-6 lg:px-8">
+        <TabNav>
+          <TabNavLink to={`/database/${database.id}`}>Database</TabNavLink>
+          <TabNavLink to={`/database/${database.id}/apps`}>Apps</TabNavLink>
+          <TabNavLink to={`/database/${database.id}/env`} selected>
```
Comment (pradel) [inline client/src/pages/database/env.tsx:43]: do we need this one? We can't set any env for the database no?

**Comment by Akirtovskis** [comment]
@pradel thanks for feedbacks, all fixed! 🚀

**Comment by pradel** [comment]
Nice good job!

**Pull request merged**